circle.yml

```diff
@@ -0,0 +1,24 @@
+---
+version: 2
+jobs:
+  build:
+    working_directory: /go/src/github.com/resin-io/sshproxy
+    docker:
+      - image: golang:1.8.3-stretch
```

Comment:
I believe the repo should have a Dockerfile that will 
1. define the dependencies of the build process
2. create the build artifacts (and most probably store them in a bind mount directory e.g. `build`). 

That way, if someone wants to rebuild `sshproxy` during development they won't have to 
1. install these manually on their host operating system
2. make sure that their golang version is in sync with this
3. Ideally, during local dev I'd be able to specify which artifact I want to build to avoid building everything (e.g. for the backend 99% of the time it'd be linux_amd64)

To give an example here's what I used to test sshproxy locally

```
# build.sh
SSHPROXY_ARTIFACT=build/linux_amd64/sshproxy

docker run --rm \
	-v $(pwd)/build:/go/src/github.com/resin-io/sshproxy/build \
	-v $(pwd)/sshproxy.go:/go/src/github.com/resin-io/sshproxy/sshproxy.go \
	sshproxy /bin/sh -c "make clean; make $SSHPROXY_ARTIFACT"
```

```
# Dockerfile

RUN apt-get update \
	&& apt-get install -y \
		bzip2 \
	&& rm -rf /var/lib/apt/lists/*

RUN mkdir -p /go/src/github.com/resin-io/sshproxy
WORKDIR /go/src/github.com/resin-io/sshproxy

COPY . .

RUN go get ./...
```